contents/teams/replay/objectives.mdx

```diff
@@ -1,24 +1,18 @@
-### 🎯 Goal 1: Quality and Consolidation for Session Replay (web & mobile)
+### 🎯 Goal 1: People think of PostHog as a mobile solution
 
-Why? 
+* Mobile replay for React Native
+* Lots of content on how to use PostHog on Mobile
 
-We've done a lot of investigation and preparation work and we want to capitalize on that
+### 🎯 Goal 2: Error tracking in people's hands
 
-* Mobile replay out of beta for Android/iOS
-* Make capture and playback as complete, performant and easy as possible
-* Build the best UX for finding and playing recordings
+* the top 5 features implemented
+* real customers using it and giving us feedback
+* integrated with CDP for notifications
```

Comment:
is notifications the new alerting?

Reply:
```suggestion
* integrated with CDP for notifications/alerting
```

i guess they're inter-changeable for me... an alert is a notification you really care about receiving in a timely manner